schema/spacewalk/upgrade/susemanager-schema-5.0.4-to-susemanager-schema-5.0.5/001-oval-add-tables.sql

```diff
@@ -34,4 +34,4 @@ CREATE TABLE IF NOT EXISTS suseOVALPlatformVulnerablePackage
 );
 
 ALTER TABLE rhnServer
-    ADD COLUMN IF NOT EXISTS cpe VARCHAR;
\ No newline at end of file
+    ADD COLUMN IF NOT EXISTS cpe VARCHAR(64) NOT NULL;
```

Comment:
AFAIK 5.0.5 is already released (Uyuni).
You need a new schema migration and cannot change the old one.